cypress.json

```diff
@@ -1,6 +1,7 @@
 {
   "excel": "https://docs.google.com/spreadsheets/d/1Yozpcv58-etk1UNvxS0HxMqw7V5Amf4dgWBnekJcHcc/edit?usp=sharing",
   "csv": "https://raw.githubusercontent.com/thoughtworks/build-your-own-radar/master/spec/end_to_end_tests/resources/sheet.csv",
+  "json": "https://raw.githubusercontent.com/yzalvin/build-your-own-radar/master/spec/end_to_end_tests/resources/data.json",
```

Comment:
This line refers to my fork to get the data.json, but it should refer to thoughtworks/build-your-own-radar, after this has been merged